docs/source/membership/membership.md

```diff
@@ -1,341 +1,197 @@
+
+In order to use the Node OU roles, the "identity classification" feature must be enabled for the network. When using the folder-based MSP structure, this is accomplished by enabling "Node OUs" in the config.yaml file which resides in the root of the MSP folder:
+
+```
+NodeOUs:
+  Enable: true
+  ClientOUIdentifier:
+    Certificate: cacerts/ca.sampleorg-cert.pem
+    OrganizationalUnitIdentifier: client
+  PeerOUIdentifier:
+    Certificate: cacerts/ca.sampleorg-cert.pem
+    OrganizationalUnitIdentifier: peer
+  AdminOUIdentifier:
+    Certificate: cacerts/ca.sampleorg-cert.pem
+    OrganizationalUnitIdentifier: admin
+  OrdererOUIdentifier:
+    Certificate: cacerts/ca.sampleorg-cert.pem
+    OrganizationalUnitIdentifier: orderer
+```
+
+In the example above, there are 4 possible Node OU `ROLES` for the MSP:
+
+   * client
+   * peer
+   * admin
+   * orderer
+
+This convention allows you to distinguish MSP roles by the OU present in the CommonName attribute of the X509 certificate. The example above says that any certificate issued by cacerts/ca.sampleorg-cert.pem which has OU=client will identified as a client, OU=peer as a peer, etc. Starting with Fabric v1.4.3, there is also an OU for the orderer and for admins. The new admins role means that you no longer have to explicitly place certs in the admincerts folder of the MSP directory. Rather, the `admin` role present in the user's signcert qualifies the identity as an admin user.
+
+These Role and OU attributes are assigned to an identity when the Fabric-CA or SDK is used to `register` a user with the CA. It is the subsequent `enroll` user command that generates the certificates in the users's /msp folder.   
+
+![MSP1c](./ca-msp-visualization.png)
+
+The resulting ROLE and OU attributes are visible inside the X.509 signing certificate located in the /signcerts folder. The `ROLE` attribute is identified as `hf.Type` and  refers to an actor's role within its organization, (specifying, for example, that an actor is a `peer`). See the following snippet from a signing certificate shows how the Roles and OUs are represented in the certificate.
+
+![MSP1d](./signcert.png)
+
+Finally, though this is a slight misuse of OUs, they can sometimes be used by different organizations in a consortium to distinguish each other. In such cases, the different organizations use the same Root CAs and Intermediate CAs for their chain of trust, but assign the `OU` field to identify members of each organization.
+
+## MSP Structure
+
+Let's explore the MSP elements that render the functionality we've described so far.
+
+A local MSP folder contains the following sub-folders:
+
+![MSP6](./membership.diagram.6.png)
+
+*The figure above shows the subfolders in a local MSP on the file system*
+
+* **config.yaml:**  Used to configure the identity classification feature in Fabric by enabling "Node OUs" and defining the accepted roles.
```

Comment:
I understand that you are trying to match the image, but I think this whole section would read better if you were trying to rank by order of importance

- cacerts
- intermediate certs
- signcerts
- keystore
- admincerts
- config.yaml
- tlscacerts
- intermediatetls

Reply:
I was trying to match the order they appear when you view the folder, but I can see your point. We can try that. I updated the image as well so the list and the image match.